spec/factories/stats.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :stat do
+    association :user, factory: :user, strategy: :build
+    type 'Stat::AnimeGenreBreakdown'
+    stats_data { { total: 0 } }
```

Comment:
first one is uhh... wrong? I have tried with the second one, and there is a default set already to `{}` at a database level.